.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash -l {0}
```

Comment:
Could you please add a comment describing the purpose of setting the default shell here? Do the tests work if you simply remove this whole `defaults` block?